packages/snjs/lib/models/app/tag.ts

```diff
@@ -51,7 +64,22 @@ export class SNTag extends SNItem {
 }
 
 export class TagMutator extends ItemMutator {
+  get typedContent(): TagContent {
+    return this.content as TagContent;
+  }
+
   set title(title: string) {
-    this.content!.title = title;
+    this.typedContent.title = title;
```

Comment:
@moughxyz this is the side effect I was talking about,
we rely on a "distant side effect", someone could define:

```
get typedContent(): TagContent {
  return {...this.content} as TagContent; // or something more relevant
}
```

and break this code without breaking the interfaces.